src/manifest.json

```diff
@@ -1,34 +1,27 @@
 {
-   "manifest_version": 2,
-    "name": "Transfer Send for Firefox",
-    "version": "1.8",
-    "description": "Facilitate file transfer with Firefox Send (https://send.firefox.com) while Keeping your Personal Information Private",
+    "manifest_version": 2,
+    "name": "Firefox Send Sidebar",
+    "version": "1.9",
```

Comment:
This `version` value will need to be bumped if the the other [existing PR](https://github.com/hellosct1/firefox-send/pull/4) is merged first.

Reply:
Actually on reviewing the PR it just adds the license file and doesn't bump the version. Please ignore the above.